app/models/energy/construction.py

```diff
@@ -437,178 +493,153 @@ class EnergyWindowMaterialBlind(BaseModel):
         ' plastic slats. The default value is 0.'
     )
 
-    front_infrared_hemispherical_emissivity: float = Schema(
+    front_infrared_hemispherical_emissivity: float=Schema(
```

Comment:
Let's just call this property `emissivity` for consistency with all of the other material names.